bedrock/firefox/views.py

```diff
@@ -81,8 +77,10 @@ def installer_help(request):
         else:
             context['installer_channel'] = installer_channel
 
-    if variant == 'b':
-        template = 'firefox/installer-help-b.html'
+    if lang_file_is_active('firefox/installer-help-redesign', locale):
```

Comment:
Suggest setting this .lang file explicitly in the template using `{% add_lang_files "firefox/installer-help-redesign" %}`. This way if the template location ever moves, the translations don't break.